packages/url-loader/src/constants/crop.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from "zod";
+
+const _cropModesSchema = z.enum([
+  "auto",
+  "fill",
+  "lfill",
+  "fill_pad",
+  "crop",
+  "thumb",
+  "scale",
+  "fit",
+  "limit",
+  "mfit",
+  "pad",
+  "lpad",
+  "mpad",
+  "imagga_scale",
+  "imagga_crop",
+]);
+
+const { _output } = _cropModesSchema;
```

Comment:
given that all of these fixes look somewhat the same, aside from the naming, is there a way to make this more concise into a helper function or something?

Reply:
The requirements to force TS to output the full expanded type are quite particular, but I can investigate a bit and see if there's a way to force it to infer some of this with a helper. There will still be some ceremony involved, but still much better than rewriting the build output like we were before.